scripts/tornadoVMInstaller.sh

```diff
@@ -104,9 +104,9 @@ function downloadCorretto17() {
 function downloadMandrel11() {
     platform=$(getPlatform)
     if [[ "$platform" == 'linux' ]]; then
-        wget https://github.com/graalvm/graalvm-ce-builds/releases/download/vm-22.1.0/graalvm-ce-java11-linux-amd64-22.1.0.tar.gz
-        tar xf mandrel-java11-linux-amd64-22.1.0.0-Final.tar.gz
-        export JAVA_HOME=$PWD/mandrel-java11-22.1.0.0-Final
+        wget https://github.com/graalvm/graalvm-ce-builds/releases/download/vm-22.2.0/graalvm-ce-java11-linux-amd64-22.2.0.tar.gz
```

Comment:
This is mistaken, it should point to Mandrel instead of GraalVM.
```
wget https://github.com/graalvm/mandrel/releases/download/mandrel-22.2.0.0-Final/mandrel-java11-linux-amd64-22.2.0.0-Final.tar.gz
```

Reply:
Yes, good catch.